backend/spec/features/admin/stock_items_spec.rb

```diff
@@ -19,4 +21,10 @@
     end
     expect(page).to have_content("Created successfully")
   end
+
+  scenario 'searching by variant' do
+    visit spree.admin_stock_items_path
+    fill_in 'SKU or Option Value', with: 'Ruby'
+    click_on 'Filter Results'
```

Comment:
This spec could also use some expanding prior to merging but it is enough to reproduce the bug.

Reply:
Yeah, we would need some kind of `expect`ation of what we see on the screen, but for now it's good enough.